pcdsdevices/epics_motor.py

```diff
@@ -1011,6 +1010,15 @@ def home(self, direction=None, wait=True, **kwargs):
         return status
 
 
+class OldBeckhoffAxisPLC(BeckhoffAxisPLC):
+    cmd_home = None
+
+
+class OldBeckhoffAxis(BeckhoffAxis):
+    plc = Cpt(OldBeckhoffAxisPLC, ':PLC:', kind='normal',
+              doc='PLC error handling.')
```

Comment:
I don't mind having these, just add docstrings that note that they are for lcls-twincat-motion prior to v1.4.0 (pre-Nov2020) and we should also override the `home` method appropriately

Reply:
Consider a name like `BeckhoffAxis_Pre140` or something to be more explicit.

Ophyd also supports device class versioning for scenarios like this, but it's probably overkill.